engineering_tools/vector_map_scripts/osm_transform.py

```diff
@@ -60,7 +60,7 @@
 
 # TODO Add the desired rotation of the output map here
 # === Fixed rotation angle ===
-rotation_deg = 0  # Counter-clockwise
```

Comment:
dont change the default values based on your case. Best to keep everything at 0, (even in my own PR)